templates/go/go.mod.twig

```diff
@@ -0,0 +1,3 @@
+module github.com/{{sdk.gitUserName}}/sdk-for-go
```

Comment:
I cannot change that one, since then the go.mod file is not generated correctly for the test-cases (here the `gitRepoName` is `reponame` for all languages.

Reply:
Happy to do it otherwise. But for now, I don't know how. @abnegate  any idea?

Reply:
@phaus For test in `tests.go`	 line 8 instead of  `"github.com/appwrite/sdk-for-go/appwrite"` you could use `"github.com/appwrite/reponame/appwrite"`

would that work?